common/src/main/java/com/wynntils/features/overlays/StopwatchFeature.java

```diff
@@ -0,0 +1,67 @@
+package com.wynntils.features.overlays;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.OverlaySize;
+import com.wynntils.core.features.overlays.TextOverlay;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyBind;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import org.lwjgl.glfw.GLFW;
+
+@ConfigCategory(Category.OVERLAYS)
+public class StopwatchFeature extends Feature {
+    @RegisterKeyBind
+    private final KeyBind toggleStopwatchKeybind =
+            new KeyBind("Toggle Stopwatch", GLFW.GLFW_KEY_KP_0, true, this::toggleStopwatch);
+
+    @RegisterKeyBind
+    private final KeyBind resetStopwatchKeybind =
+            new KeyBind("Reset Stopwatch", GLFW.GLFW_KEY_KP_DECIMAL, true, Models.Stopwatch::reset);
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
```

Comment:
This might be a good place to use the new command support:
```
    @RegisterCommand
    private final LiteralCommandNode<CommandSourceStack> startCommand =
            Commands.literal("start").executes(Models.Stopwatch::start).build();

    @RegisterCommand
    private final LiteralCommandNode<CommandSourceStack> startCommand =
            Commands.literal("stop").executes(Models.Stopwatch::stop).build();

    @RegisterCommand
    private final LiteralCommandNode<CommandSourceStack> resetCommand =
            Commands.literal("reset").executes(Models.Stopwatch::reset).build();
```

That's all there is to it, I think.